README.md

```diff
@@ -168,19 +162,28 @@ Crawler has template configuration files that contain every configuration availa
 - [config/crawler.yml.example](config/crawler.yml.example)
 - [config/elasticsearch.yml.example](config/elasticsearch.yml.example)
 
-To use these files, make a copy in the same directory without the `.example` suffix:
+To use these files, make a copy locally without the `.example` suffix.
+Then remove the `#` comment-out characters from the configurations that you need.
+
+You can then copy the file into your running Docker image.
 
 ```bash
-$ cp config/crawler.yml.example config/crawler.yml
+$ docker cp config/my-crawler.yml crawler:app/config/my-crawler.yml
```

Comment:
I wonder - what is our position on copying configs vs mounting volumes?

Connectors require mounting, while crawler copying

Reply:
I hadn't considered this. Let's discuss as a team and if mounting is better we can create an issue to update docs.